app/utils/responseHandler.ts

```diff
@@ -0,0 +1,16 @@
+import { Response } from "express";
+
+export const successResponse = (res: Response, statusCode: number, status: boolean, message: string, data: Record<string, unknown> | null = {}) => {
+  res.status(statusCode).json({
+    success: status,
```

Comment:
why is it success here but the function argument is status, success sounds better